### DISCUSSION TITLE
Cloud Drive comparing - like OneDrive 2 Dropbox

### DISCUSSION THREAD

**Discussion by igiqr** [WinMerge/winmerge#1195] [Ideas]
Is there a possibility to compare folders in cloud drives. It would be good to avoid installation of their sync programs on the computer.

**Comment by sdottaka** [comment]
Unfortunately, WinMerge can only compare folders on the (real | virtual) disk to which the drive letter is assigned or shared folders such as `\\ computername\...`.
